portfolio/pom.xml

```diff
@@ -42,7 +46,6 @@
         <artifactId>appengine-maven-plugin</artifactId>
         <version>2.2.0</version>
         <configuration>
-          <!-- [DONE]: set project ID. -->
```

Comment:
Can you please rebase this branch from master? I believe this comment is removed in master and should not show up in this pull request.

Reply:
okay